tests/acceptance/conditions/test_offchain_evaluation.py

```diff
@@ -0,0 +1,206 @@
+import json
```

Comment:
Is it possible to move this module to `tests/unit/conditions`? Is there something in the `acceptance` fixtures that's needed? 

If possible to move, it'll improve the test running time by skipping the excessive setup that acceptance test fixtures perform.

Reply:
✔️